net.worldofpadman.WoP.yaml

```diff
@@ -0,0 +1,56 @@
+app-id: net.worldofpadman.WoP
+runtime: org.freedesktop.Platform
+runtime-version: 22.08
+sdk: org.freedesktop.Sdk
+command: worldofpadman
+finish-args:
+  # Controller support
+  - --device=all
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
```

Comment:
does it really support wayland?

Reply:
It does. :-) I have tested this on my F37 system with x11/fallback-x11 sockets disabled.